File: app/src/main/java/com/tonitealive/app/ui/views/signin/SignInActivity.java
package com.tonitealive.app.ui.views.signin;

import android.support.v4.app.Fragment;

import com.tonitealive.app.ui.views.SingleFragmentActivity;


public final class SignInActivity extends SingleFragmentActivity {

    @Override
    protected Fragment createFragment() {
        return SignInFragment.newInstance();
    }

}

File: app/src/main/kotlin/com/tonitealive/app/ui/views/signin/SignInActivity.kt
package com.tonitealive.app.ui.views.signin

import android.support.v4.app.Fragment
import com.tonitealive.app.ui.views.SingleFragmentActivity


class SignInActivity : SingleFragmentActivity() {

    override fun createFragment(): Fragment {
        return SignInFragment.newInstance()
    }
}